src/client/sandbox/node/element.js

```diff
@@ -161,6 +162,40 @@ export default class ElementSandbox extends SandboxBase {
         var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
         var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
         var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
+        var overrideInsertBefore         = (newNode, refNode, parentNode) => {
```

Comment:
Why it's implemented here as arrow instead of `this.overridedMethods.insertBefore`?